test/clustertest/tests/TimingTest.cpp

```diff
@@ -3,26 +3,16 @@
 
 struct TimingTest : tpunit::TestFixture {
     TimingTest()
-        : tpunit::TestFixture("TimingTest",
-                              BEFORE_CLASS(TimingTest::setup),
-                              AFTER_CLASS(TimingTest::teardown),
-                              TEST(TimingTest::test)) { }
+        : tpunit::TestFixture("Timing", TEST(TimingTest::test)) { }
 
     BedrockClusterTester* tester;
```

Comment:
We could, but I really don't want to clean up all the tests in this PR if I don't need to. I only fixed the names because I was annoyed trying to run individual tests and the naming was so inconsistent.

Reply:
oh yeah that's fair I only meant the `BedrockClusterTester* tester` here b/c it's redefined in the test method now.

Reply:
Oh, oops, yes, I'll fix this one.